internal/controllers/server/tests/create-adoption/01-assert.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Server
+metadata:
+  name: adoption
+status:
+  resource:
+    name: adoption
```

Comment:
Nice. Matching on name avoids having to work out how to check the ID's don't match.